pattern-1/volumes/mysql-scripts/mysql-wso2apim-analytics-db.sql

```diff
@@ -0,0 +1,229 @@
+CREATE DATABASE IF NOT EXISTS wso2apim_analytics_db;
+USE wso2apim_analytics_db;
+
+CREATE TABLE IF NOT EXISTS REG_CLUSTER_LOCK (
+  REG_LOCK_NAME VARCHAR (20),
```

Comment:
Why are we creating `REG` tables for the analytics db? 

For API-M Analytics, we only need to create the database, the tables will  be populated when the spark scripts are executed.

Reply:
@msmshariq, I think this is for the registry-db of Analytics and not for event store or processed store dbs, those are the ones that's auto-generated, Isn't that the case here ?

Reply:
Yes, that's what I also thought. Each WSO2 product ships a collection of database scripts to be executed at the installation time for creating the databases and tables required. If that's not the case we might need to remove them from the product distribution.